src/components/pages/signUp/index.tsx

```diff
@@ -45,95 +52,90 @@ export default function SignUpPage(): ReactElement {
         <HStack style={{ height: '100%' }}>
             <VStack style={{ flex: 1, padding: '32px', overflowY: 'scroll' }}>
                 <VStack style={{ gap: '32px', width: '100%', maxWidth: '800px' }}>
-                    <HStack style={{ width: '100%' }}>
+                    <HStack style={{ width: '100%', maxWidth: '400px' }}>
                         <Image src={'/people-eat-logo-title.png'} alt="" width={203} height={46} />
                         <Spacer />
                     </HStack>
 
-                    <VStack style={{ width: '100%', alignItems: 'flex-start' }}>
-                        <h2>Find a private chef!</h2>
-                        <p>Please enter your details</p>
+                    <VStack style={{ width: '100%', maxWidth: '400px', alignItems: 'flex-start', marginTop: '100px' }}>
+                        <h2 className={'text-heading-xl my-1'}>Find a private chef!</h2>
+                        <p className={'text-preBlack my-1'}>Please enter your details</p>
                     </VStack>
 
                     <HStack style={{ gap: '32px' }}>
-                        <PEIconButton icon={Icon.apple} />
-                        <PEIconButton icon={Icon.google} />
+                        <PEIconButton icon={Icon.appleWhiteIcon} size={'54px'} bg={'#000'} iconSize={24} />
+                        <PEIconButton icon={Icon.facebook} size={'54px'} bg={'#3B5998'} iconSize={48} />
+                        <PEIconButton icon={Icon.google} size={'54px'} bg={'#F5F5F5'} iconSize={24} />
                     </HStack>
 
-                    <HStack style={{ width: '100%', alignItems: 'center', gap: '32px' }}>
+                    <HStack style={{ width: '100%', maxWidth: '400px', alignItems: 'center', gap: '32px' }}>
                         <div style={{ height: '1px', backgroundColor: '#F5F5F5', flex: 1 }}></div>
                         <p>with</p>
                         <div style={{ height: '1px', backgroundColor: '#F5F5F5', flex: 1 }}></div>
                     </HStack>
 
-                    <VStack style={{ width: '100%', alignItems: 'flex-start' }}>
+                    <VStack style={{ width: '100%', maxWidth: '400px', alignItems: 'flex-start' }}>
                         <p>First Name</p>
-                        <TextField
-                            style={{ width: '100%' }}
-                            value={firstName}
-                            onChange={(event): void => {
-                                setFirstName(event.target.value);
-                            }}
-                            variant="outlined"
-                        />
+                        <PEInput value={firstName} onChange={setFirstName} type={'text'} placeholder={'firstname'} />
```

Comment:
This should be two words in english

```suggestion
                        <PEInput value={firstName} onChange={setFirstName} type={'text'} placeholder="First name" />
```